website/docs/r/log_profile.html.markdown

```diff
@@ -0,0 +1,104 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_log_profile"
+sidebar_current: "docs-azurerm-resource-monitor-log-profile"
+description: |-
+  Creates a Log Profile.
+---
+
+# azurerm_log_profile
+
+Creates a [Log Profile](https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitoring-overview-activity-logs#export-the-activity-log-with-a-log-profile). A Log Profile configures how Activity Logs are exported.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "logprofiletest-rg"
+  location = "eastus"
+}
+
+resource "azurerm_storage_account" "test" {
+  name                     = "afscsdfytw"
+  resource_group_name      = "${azurerm_resource_group.test.name}"
+  location                 = "${azurerm_resource_group.test.location}"
+  account_tier             = "Standard"
+  account_replication_type = "GRS"
+}
+	
+resource "azurerm_eventhub_namespace" "test" {
+  name                = "logprofileeventhub"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  sku                 = "Standard"
+  capacity            = 2
+}
+
+resource "azurerm_log_profile" "test" {
+  name = "default"
+
+  categories = [
+    "Action",
+    "Delete",
+    "Write",
+  ]
+	
+  locations = [
+    "centralus",
+    "eastus",
+    "eastus2",
+    "westus",
+    "northcentralus",
+    "southcentralus",
+    "global",
```

Comment:
for the purposes of this example we should be able to simplify this to just `global`?